docfx.json

```diff
@@ -320,6 +320,22 @@
         "articles/governance/blueprints/**/*.md": "Azure Blueprints",
         "articles/governance/policy/**/*.md": "Azure Policy",
         "articles/governance/resource-graph/**/*.md": "Azure Resource Graph"
+      },
+      "featureFlags": {
+        "articles/**/*.md": [
+          "show_learn_banner"
+        ]
+      },
+      "learn_banner_products": {
+        "articles/azure-virtual-machines/**/*.md": [
```

Comment:
@shirgoldbird if you want to provide Azure as the default product you should include this line: 

```
"learn_banner_products": {
    "articles/**/*.md": [
       "azure"
    ],
    ...
```